modules/webrtc_aec/aec.cpp

```diff
@@ -31,6 +31,7 @@
 
 using namespace webrtc;
 
+static bool webrtc_aec_extended_filter;
```

Comment:
Shouldn't this be added to the struct aec?

Reply:
I used the implementation of `vumeter_stderr` from `modules/vumeter/vumeter.c` as template, which uses a static, global variable.

I would also prefer to avoid the global variable, but i'm not sure, how to access `aec` within `module_init`.